### PR TITLE
feat: Improve localization translations

### PR DESCRIPTION
**Pull request by ashchan**

### CODE REVIEW

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -210,18 +209,6 @@
     "footer": {
       "fail-to-fetch-tip-block-number": "Cannot fetch tip block number"
     },
-    "menuitem": {
```
Comment (ashchan) [inline packages/neuron-ui/src/locales/en.json]: @Keith-CY I guess we don't use these any more. Can you check and confirm?

Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json]: Yes, they are not used any more.

Comment (ashchan) [reply packages/neuron-ui/src/locales/en.json]: Nice. Thanks!

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -243,10 +230,10 @@
     "messages": {
       "error": "Error",
       "unknown-error": "Unknown error",
-      "update-wallet-successfully": "Update the wallet successfully",
-      "delete-wallet-successfully": "Delete the wallet successfully",
-      "create-network-successfully": "Create a network successfully",
-      "update-network-successfully": "Update the network successfully",
+      "update-wallet-successfully": "The wallet was updated",
+      "delete-wallet-successfully": "The Wallet was deleted",
```
Comment (classicalliu) [inline packages/neuron-ui/src/locales/en.json]: `Wallet` or `wallet` ?

Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json]: I think it should be `wallet`

**Pull request merged**